### DISCUSSION TITLE
After running run.bat or run_anime.bat it just loads and pauses right away.

### DISCUSSION THREAD

**Discussion by MathTheNight** [lllyasviel/Fooocus#3249] [Q&A] [answered]
I'm not sure if it's already been reported or if I'm missing some valuable information in the console to understand but basically it worked at first but ever since I downloaded the files from run_anime.bat it just started not working anymore and whenever I'd execute the file it'd just end up pausing by itself.

```
C:\Users\Lenovo\Desktop\AI>.\python_embeded\python.exe -s Fooocus\entry_with_update.py --preset anime
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py', '--preset', 'anime']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.4.3
Loaded preset: C:\Users\Lenovo\Desktop\AI\Fooocus\presets\anime.json
[Cleanup] Attempting to delete content of temp dir C:\Users\Lenovo\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Total VRAM 6144 MB, total RAM 8111 MB
Set vram state to: NORMAL_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce GTX 1660 Ti : native
VAE dtype: torch.float32
Using pytorch cross attention
Refiner unloaded.

C:\Users\Lenovo\Desktop\AI>pause
Press any key to continue . . .
```

**Comment by omda72** [comment]
I had the same experience , and unfortunately no body is helping , after it mention '' press any key to continue'' the console disappear.

**Comment by stanhp** [comment]
Hello, did you tried to start the run.bat then change the setting to anime ? It use to change the parameter to the same as run anime .

Envoyé de mon iPhone

**Comment by mashb1t** [comment, marked as answer]
Ensure you are fulfilling the minimum system specifications listed in the readme as well as your swap setup is correct (see [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md)).

**Reply by MathTheNight** [reply]
Yes, I apologize. I've been confused and was wondering whether my computer can process it, I underestimated it. 
It also helped me fix it by restarting as there were too many processes cluttering my system at that time.

**Comment by omda72** [comment]
That's the basic answer , my system is as follow:
Intel i7 11generation
16 RAM
512 SSD

________________________________

**Reply by mashb1t** [reply]
If you do not have a dedicated GPU in your laptop you have to edit the run.bat and add `--always-cpu` at the end of the command (line 1, before pause)

**Comment by omda72** [comment]
I did all the options , whether to press run
Or realistic run
Or animation run
But nothing worked
....uninstall and installed again
....remove webshield and security measures
...stop the firewall
Nothing worked ....do I have to download python language to my laptop
Thanks a lot
________________________________

**Comment by omda72** [comment]
Ok , thanks for your consideration

Sent from my Bell Samsung device over Canada’s largest network.
________________________________

**Comment by omda72** [comment]
Hi there , do I need to restart laptop.
________________________________

**Reply by MathTheNight** [reply]
If your system is running for a while, then restarting it might give you more process space as you potentially get rid of any active background apps that take a small chunk of process in a huge amount.

**Comment by Buddy533** [comment]
Set the swap file to 25gb and you'll be fine.

**Comment by omda72** [comment]
How to set the swap file to 25 GB coz I tried to edit run.bat and added --always-CPU and still no clue ..
Thanks for following on me

Sent from my Bell Samsung device over Canada’s largest network.
________________________________

**Reply by mashb1t** [reply]
see [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#system-swap), set swap manually instead of automatic allocation

**Comment by omda72** [comment]
Unfortunately, tried manual change in RAM  to 2034 MB and nothing happen , it look like this software works only on python friendly PC.
Thanks for you help
________________________________

**Reply by mashb1t** [reply]
Please set swap to 30 GB on the same drive you've put Fooocus on and ensure you have that much free disk apace. It doesn't matter if your PC is "python friendly" (whatever this means) but it only matters that swap is set up correctly and the minimum requirements are fulfilled.

**Comment by omda72** [comment]
Tried manually changing the RAM or allocated memory to 2034 MB but nothing has changed, it look like this software donèt work unless the pc is python friendly or having AMD processor, anyhow thanks for your help and concern.

________________________________

**Comment by omda72** [comment]
Ok ,  I will do that .👍

Sent from my Bell Samsung device over Canada’s largest network.
________________________________
